### PR TITLE
Fix tentacle config display

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/269

### CODE REVIEW

File: Services/Interfaces/web_interface/models/configuration.py [hunk 1]
```diff
@@ -41,8 +42,7 @@
 from octobot_evaluators.evaluator.abstract_evaluator import AbstractEvaluator
 from octobot_backtesting.api.backtesting import is_backtesting_enabled
 from octobot.community.community_manager import CommunityManager
-from octobot_tentacles_manager.api.configurator import save_tentacles_setup_configuration
-from octobot_tentacles_manager.api.inspector import get_tentacle_resources_path
+from octobot.constants import CONFIG_FILE_SCHEMA
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/configuration.py:45]: I don't know what to think about this import. Do we really want to import something directly in OctoBot's package ?

Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/configuration.py:45]: We don't really have a choice most tentacle import things from the OctoBot package (constants, tests, etc)

Comment (Herklos) [reply Services/Interfaces/web_interface/models/configuration.py:45]: Oh ok :+1:

File: Services/Interfaces/web_interface/models/configuration.py [hunk 2]
```diff
@@ -184,6 +204,42 @@ def get_tentacle_config_schema(klass):
...
+    startup_tentacles_activation = get_tentacles_activation(get_startup_tentacles_config())
+    activation_by_group = {}
+    for root_element in NON_TRADING_STRATEGY_RELATED_TENTACLES:
+        try:
+            _add_tentacles_activation_desc_for_group(activation_by_group, tentacles_activation,
+                                                     startup_tentacles_activation, root_element, media_url)
+        except KeyError:
+            pass
+    # only return tentacle groups for which there is an activation choice to simplify the config interface
+    return {group: tentacles
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/configuration.py:238]: :+1:

File: Services/Interfaces/web_interface/models/configuration.py [hunk 3]
```diff
@@ -76,22 +81,35 @@
...
-            return doc_content.replace("resources/", resource_url)
+def get_tentacle_documentation(name, media_url):
+    try:
+        doc_file = get_tentacle_documentation_path(name)
+        if isfile(doc_file):
+            resource_url = f"{media_url}/{get_tentacle_resources_path(name).replace(sep, '/')}/"
+            with open(doc_file) as doc_file:
+                doc_content = doc_file.read()
+                # patch resources paths into the tentacle resource path
+                return doc_content.replace("resources/", resource_url)
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/configuration.py:108]: yes !

**Pull request merged**